interpreter/CMakeLists.txt

```diff
@@ -286,8 +286,10 @@ else()
   include(VersionFromVCS)
 
   set(PACKAGE_VERSION "${LLVM_PACKAGE_VERSION}")
-  if (NOT (PACKAGE_VERSION VERSION_EQUAL 5.0 OR PACKAGE_VERSION VERSION_EQUAL 5.0.1))
-    message(FATAL_ERROR "LLVM version different from 5.0 or 5.0.1!")
+  if (${PACKAGE_VERSION} MATCHES "5\\.0(|\\.[1-2])")
```

Comment:
5.0.x should be API compatible. Let's do .*

Reply:
thanks, it is good remark! I forgot about ROOT patched version which is 5.0.0! Fixed!